packages/back-end/src/models/FactMetricModel.ts

```diff
@@ -175,6 +175,8 @@ export class FactMetricModel extends BaseClass {
 
     return {
       ...otherFields,
+      riskThresholdDanger: otherFields.loseRisk,
+      riskThresholdSuccess: otherFields.winRisk,
```

Comment:
This will result in both `loseRisk` and `riskThresholdDanger` being added to the object.

Instead, add `loseRisk` and `winRisk` to the destructure call above like we do for `metricType` for example.

Reply:
Ah shoot. ty